cmd/loadtest/loadtest.go

```diff
@@ -30,6 +33,20 @@ func RunLoadTestCmdF(cmd *cobra.Command, args []string) error {
 
 	controllerType := config.UserControllerConfiguration.Type
 
+	s := os.Getenv("MM_LOADTEST_SEED")
+	var seed int64
+	if s != "" {
+		v, err := strconv.Atoi(s)
+		if err != nil {
+			mlog.Error(fmt.Sprintf("could not convert %q to a numeric value", s))
```

Comment:
I think we should exit the program with error if this happens. No need to continue.

Reply:
Was not so sure about this. Anyways, it's done 👍